CMakeLists.txt

```diff
@@ -81,6 +81,7 @@ if(APPLE)
 	set_property(DIRECTORY
 		APPEND
 		PROPERTY COMPILE_DEFINITIONS SC_DARWIN)
+	set(CMAKE_INSTALL_PREFIX "${CMAKE_BINARY_DIR}/Install")
```

Comment:
I was still wondering about this PR. IMO should not have merged so fast.

so as it is this is now installing to {project dir}/Install ?

and a user would either use it in place or copy it to somewhere.


On Sun, Jun 28, 2015 at 11:56 AM Rainer Schütz <notifications@github.com>
wrote:

> In CMakeLists.txt
> <https://github.com/supercollider/supercollider/pull/1569#discussion_r33421148>
> :
>
> > @@ -81,6 +81,7 @@ if(APPLE)
> >  	set_property(DIRECTORY
> >  		APPEND
> >  		PROPERTY COMPILE_DEFINITIONS SC_DARWIN)
> > +	set(CMAKE_INSTALL_PREFIX "${CMAKE_BINARY_DIR}/Install")
>
> @scztt <https://github.com/scztt>, sorry for extending this thread a bit:
> running sc in Debug mode from the IDE doesn't seem exactly
> self-explanatory. Is it the best way to "build" the "SuperCollider" Scheme
> rather than the all_builds target?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/supercollider/supercollider/pull/1569/files#r33421148>
> .
>